### PR TITLE
Modify RSKIP134

### PR DESCRIPTION
**Pull request by josedahlquist**
Added a missing validation to the RSKIP

### CODE REVIEW

File: IPs/RSKIP134.md [hunk 1]
```diff
@@ -12,55 +12,55 @@
 
 ## Abstract
 
-RSK has a two way peg system to lock funds in the Federation, using BTC, that are then released as RBTC in RSK through the Bridge native contract. This system was implemented using a whitelisting process that adds a nuisance in the usability of the network.
+RSK has a two-way peg system to lock funds in the Federation, using BTC, which is then released as RBTC in RSK through the Bridge native contract. This system was implemented using a whitelisting process that adds a nuisance to the usability of the network.
```
Comment (marcos-iov) [inline IPs/RSKIP134.md]: ...which ~~is~~ _**are**_ then released...

File: IPs/RSKIP134.md [hunk 2]
```diff
@@ -12,55 +12,55 @@
...
 
-We will define a process that replaces the whitelisting by restricting the funds locked in the Federation at any time, taking us closer to a fully decentralized two way peg, while avoiding unnecesary risks for the network users.
+This RSKIP defines a process on the side of the whitelisting to restrict the funds locked in the Federation at any time, taking the platform closer to a fully decentralized two-way peg, while avoiding unnecessary risks for the network users.
```
Comment (marcos-iov) [inline IPs/RSKIP134.md]: ...~~on the side of the whitelisting~~ _**on the whitelisting side**_

Comment (marcos-iov) [inline IPs/RSKIP134.md]: ...to restrict the funds locked in the Federation 

Maybe change for **_to restrict the amount of funds locked in the Federation_**?

Comment (marcos-iov) [inline IPs/RSKIP134.md]: Add a full stop 

...the funds locked in the Federation at any time. Taking the platform closer...

File: IPs/RSKIP134.md [hunk 3]
```diff
@@ -12,55 +12,55 @@
...
 
 ## Motivation
 
-RSK was created as a second layer on top of Bitcoin, bringing the smart contracts capabilities to the ecosystem, increasing the overall value for the end user.
-The gateway between BTC and RSK is the two way peg, if this system is complex and troublesome, it defeat its own purpose.
+RSK was created as a second layer on top of Bitcoin, bringing the capabilities of the smart contract to the ecosystem, increasing the overall value for the end-user.
```
Comment (marcos-iov) [inline IPs/RSKIP134.md]: ...bringing the capabilities of ~~the~~ smart contract **_(s)_**

File: IPs/RSKIP134.md [hunk 4]
```diff
@@ -12,55 +12,55 @@
...
+The gateway between BTC and RSK is the two-way peg if this system is complex and troublesome, it defeats its purpose.
```
Comment (marcos-iov) [inline IPs/RSKIP134.md]: Move the comma

The gateway between BTC and RSK is the two-way peg, if this system is complex and troublesome it defeats its purpose.

**Review by SergioDemianLerner — APPROVED** [review]

**Pull request merged**